corehq/warehouse/dbaccessors.py

```diff
@@ -40,3 +42,24 @@ def _get_ids_by_last_modified(cls, doc_types, start_datetime, end_datetime):
         )
         for result in results:
             yield result['id']
+
+
+def get_synclog_ids_by_date(start_datetime, end_datetime):
+    from casexml.apps.phone.models import SyncLog
+
+    results = SyncLog.view(
+        "sync_logs_by_date/view",
+        startkey=[json_format_datetime(start_datetime)],
+        endkey=[json_format_datetime(end_datetime)],
+        reduce=False,
+        include_docs=False
+    )
+    for result in results:
+        yield result['id']
```

Comment:
right that makes sense. i was just thinking in terms of the synclog fact, but it could lead to some hard to figure out bugs. i like the idea of making it a standard. updated here: https://github.com/dimagi/commcare-hq/pull/16327/commits/f3758145405586f6855a318a031a343611b6f6b9